Gordon360/Models/ViewModels/RecIM/CreateTeamViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Gordon360.Models.CCT;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class CreateTeamViewModel
+    {
+        public string Name { get; set; }
+        public int StatusID { get; set; }
+        public int ActivityID { get; set; }
+        public bool Private { get; set; }
+        public string? Logo { get; set; }
+
+    }
+}
```

Comment:
CreateTeamViewModel will be the simplified object for the UI to pass to the API. This will allow the UI to not have to deal with manually setting a Null teamID, default statusID, (we can also set a default picture) so on and so forth.

Reply:
@EjPlatzer should we be standardizing the names of upload/create viewmodels? We currently have a few following the format "__objectname__UploadViewModel" (i.e. MembershipUploadViewModel) and I am wondering if they should be consistently named.

Reply:
I'm not sure it matters all that much. Perhaps `New[object]ViewModel` would be best, since it keeps the noun convention of other models and expresses that the model represents new data. But I don't think this is something we need to worry about strictly enforcing.

Reply:
I don't mind either, but if you guys want to commit to the New____ViewModel, then I'll make a PR later to update the recim viewmodels to follow that convention.

Reply:
I think it is very helpful from the code quality side to be consistent about it. With consistent nomenclature, it will be easy to understand the purpose of such objects and to find them in the explorer.

Reply:
If we're thinking in terms of consistency and searchability, it's probably better to have it start with the object so that it gets sorted alongside the related view models (i.e. all the Membership view models are together, rather than a massive clump of `Create[object]` or `New[object]` models unrelated to each other.

So I suppose `[object]UploadViewModel` may fit the bill best after all.